src/common/resources/pieceResources.js

```diff
@@ -0,0 +1,31 @@
+import {
+  baseManifest,
+} from '@folio/stripes-acq-components';
+
+import {
+  ORDER_PIECES_API,
+  PIECES_API,
+  CHECKIN_API,
+} from '../constants';
+
+export const pieceResource = {
+  ...baseManifest,
+  clientGeneratePk: false,
```

Comment:
should we discuss an option of adding this to the `baseManifest`?

Reply:
We use it very rarely